src/main/app/src/app/informatie-objecten/informatie-objecten.service.ts

```diff
@@ -88,30 +88,35 @@ export class InformatieObjectenService {
     zaakUuid: string,
     documentReferentieId: string,
     infoObject: EnkelvoudigInformatieobject,
-    taakobject: boolean,
+    taakObject: boolean,
   ): Observable<EnkelvoudigInformatieobject> {
     const formData = new FormData();
-    formData.append("bestandsnaam", infoObject.bestandsnaam);
-    formData.append("titel", infoObject.titel);
-    formData.append("bestandomvang", infoObject.bestandsomvang.toString());
-    formData.append("formaat", infoObject.formaat);
-    formData.append(
-      "informatieobjectTypeUUID",
-      infoObject.informatieobjectTypeUUID,
-    );
-    formData.append(
-      "vertrouwelijkheidaanduiding",
-      infoObject.vertrouwelijkheidaanduiding,
-    );
-    formData.append("status", infoObject.status);
-    formData.append(
-      "creatiedatum",
-      moment(infoObject.creatiedatum).format("YYYY-MM-DDThh:mmZ"),
-    );
-    formData.append("auteur", infoObject.auteur);
-    formData.append("taal", infoObject.taal);
-
-    formData.append("file", infoObject.bestand, infoObject.bestandsnaam);
+    if (infoObject.bestandsnaam)
+      formData.append("bestandsnaam", infoObject.bestandsnaam);
+    if (infoObject.titel) formData.append("titel", infoObject.titel);
+    if (infoObject.bestandsomvang)
+      formData.append("bestandomvang", infoObject.bestandsomvang.toString());
+    if (infoObject.formaat) formData.append("formaat", infoObject.formaat);
+    if (infoObject.informatieobjectTypeUUID)
+      formData.append(
+        "informatieobjectTypeUUID",
+        infoObject.informatieobjectTypeUUID,
+      );
+    if (infoObject.vertrouwelijkheidaanduiding)
+      formData.append(
+        "vertrouwelijkheidaanduiding",
+        infoObject.vertrouwelijkheidaanduiding,
+      );
+    if (infoObject.status) formData.append("status", infoObject.status);
+    if (infoObject.creatiedatum)
+      formData.append(
+        "creatiedatum",
+        moment(infoObject.creatiedatum).format("YYYY-MM-DDThh:mmZ"),
+      );
+    if (infoObject.auteur) formData.append("auteur", infoObject.auteur);
+    if (infoObject.taal) formData.append("taal", infoObject.taal);
+    if (infoObject.bestand)
```

Comment:
This logic is repeated over and over, can we extract it into a small helper method? 
Thinking of something along these lines:
```
appendFormData(form: FormData, key: string, from: any) {
    const value = from[key];
    if (value) {
        form.append(key, value);
    }
}
```

Does that make sense?